build/cloud/07-deployment.yaml

```diff
@@ -25,16 +25,12 @@ spec:
           mountPath: /etc/kubeedge/cloud
           subPath: kubeconfig.yaml
         command:
-        - /bin/sh
-        - -c
-        - |
-          apk --no-cache add coreutils && cat | tee /etc/kubeedge/cloud/kubeconfig.yaml <<EOF
```

Comment:
The `sh` command is still required to print the file

Reply:
I think we can delete the initContainer here, because cloudcore can use `serviceaccount that mount to pod` to connect to apiserver, thought? @johscheuer @osiriswd

Reply:
No need construct the kubeconfig manually, K8s client-go can use incluster mode. 
cc @kadisi @chendave

Reply:
Fixed. @johscheuer